airbyte-integrations/connectors/source-hubspot/source_hubspot/source.py

```diff
@@ -82,15 +82,10 @@ def get_api(config: Mapping[str, Any]) -> API:
         return API(credentials=credentials)
 
     def get_common_params(self, config) -> Mapping[str, Any]:
-        start_date = config.get("start_date")
+        start_date = config["start_date"]
         credentials = config["credentials"]
         api = self.get_api(config=config)
-        common_params = dict(api=api, start_date=start_date, credentials=credentials)
-
-        credentials_title = credentials.get("credentials_title")
-        if credentials_title == OAUTH_CREDENTIALS or credentials_title == PRIVATE_APP_CREDENTIALS:
-            common_params["authenticator"] = api.get_authenticator()
```

Comment:
the gist of the difference between this PR and https://github.com/airbytehq/airbyte/pull/15099/ is that the authenticator is now set in the stream's `__init__` so it's properly set for AssociationStream too.